duckdb-vortex/src/include/vortex_common.hpp

```diff
@@ -139,7 +139,7 @@ struct ArrayStreamSink {
 
 	~ArrayStreamSink() {
 		// "should dctor a sink, before closing it
-		D_ASSERT(sink == nullptr);
+		// D_ASSERT(sink == nullptr);
```

Comment:
can we remove this and put a longer comment or maybe a print to stderr

Reply:
Seems like the canonical thing to do in C++ is call close here if we're not closed. Also internet suggests that you shouldn't throw from C++ destructors

Reply:
But closing can also throw

Reply:
yes, that's why this is hard to do correctly